### PR TITLE
[#960][part-2] feat(dashboard): Add a dashboard front-end module.

### PR DESCRIPTION
**Pull request by yl09099**
### What changes were proposed in this pull request?

Add a UI front-end file for dashboard, which have been verified it in DIDI's production:
image
image
image

### Why are the changes needed?

Fix: #960 

### Does this PR introduce _any_ user-facing change?

Add the web dashboard for coordinator

### How was this patch tested?

UT.

### CODE REVIEW

File: deploy/kubernetes/docker/hadoopconfig/.gitkeep [hunk 1]
```diff
@@ -1,16 +0,0 @@
-#
-# Licensed to the Apache Software Foundation (ASF) under one or more
-# contributor license agreements.  See the NOTICE file distributed with
-# this work for additional information regarding copyright ownership.
```
Comment (slfan1989) [inline deploy/kubernetes/docker/hadoopconfig/.gitkeep:4]: What is the effect of this change?

File: dashboard/pom.xml [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>uniffle-parent</artifactId>
+        <groupId>org.apache.uniffle</groupId>
+        <version>0.8.0-SNAPSHOT</version>
```
Comment (jerqi) [inline dashboard/pom.xml:24]: 0.9.0-SNAPSHOT.

**Comment by jerqi** [comment]
@xianjingfeng @leixm Could you help me review this pr?

File: dashboard/src/main/webapp/src/components/LayoutPage.vue [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+                active-text-color="#ffd04b"
+                @select="handleSelect">
+              <el-menu-item index="0">
+                <div class="unffilelogo">
+                  <img src="../assets/uniffle-logo.png" alt="unffile">
+                </div>
+              </el-menu-item>
+              <router-link to="/coordinatorserverpage">
+                <el-menu-item index="1">
+                  Coordinator Server
```
Comment (zuston) [inline dashboard/src/main/webapp/src/components/LayoutPage.vue:40]: `Coordinator Server` -> `Coordinator`

File: dashboard/src/main/webapp/src/components/LayoutPage.vue [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+<script>
+import {ref, onMounted} from 'vue'
+
+export default {
+  setup() {
+    const activeIndex1 = ref('1')
+
+    function handleSelect(index, path) {
+      console.log("index", index)
```
Comment (zuston) [inline dashboard/src/main/webapp/src/components/LayoutPage.vue:72]: remove this

File: dashboard/src/main/webapp/src/components/LayoutPage.vue [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+      console.log("path", path)
```
Comment (zuston) [inline dashboard/src/main/webapp/src/components/LayoutPage.vue:73]: ditto

File: dashboard/src/main/webapp/src/components/CoordinatorServerPage.vue [hunk 6]
```diff
@@ -0,0 +1,149 @@
...
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<template>
+  <div class="demo-collapse">
+    <el-collapse v-model="pageData.activeNames" accordion:false>
+      <el-collapse-item title="Coordinator Server" name="1">
```
Comment (zuston) [inline dashboard/src/main/webapp/src/components/CoordinatorServerPage.vue:21]: This title is not necessary, all the page is under the coordinator server page.

Comment (yl09099) [reply dashboard/src/main/webapp/src/components/CoordinatorServerPage.vue:21]: > This title is not necessary, all the page is under the coordinator server page.

The page has a collapsible effect. We need to add this title here.

**Review by zuston — APPROVED** [review]
lgtm. Waiting for other people’s reviews

**Comment by zuston** [comment]
Merged. Thanks for your great contribution @yl09099

**Pull request merged**